src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.data.mongodb.host=localhost
```

Comment:
instead of using `.properties`, create a `.yaml`. It simplifies configurations that share the same structure.

It would look like this:
```yaml
spring:
    data:
        mongodb:
            host: localhost 
            port: 27017
            database: hospitalmanagementdb
```